build/cluster-operator-ansible/playbooks/cluster-operator/aws/infrastructure.yml

```diff
@@ -20,30 +20,30 @@
 
 - import_playbook: ../../aws/openshift-cluster/provision_elb.yml
 
-# - name: Create iam roles
-#   hosts: localhost
-#   roles:
-#   - openshift_aws
-#   tasks:
-#   - include_role:
-#       name: openshift_aws
-#       tasks_from: iam_role
-#     vars:
-#       l_node_group_config: "{{ openshift_aws_node_group_config }}"
-#       openshift_aws_iam_node_role_name: "openshift_node_describe_instances"
-#       openshift_aws_iam_node_role_policy_json: "{{ lookup('file', 'describeinstances.json') }}"
-#       openshift_aws_iam_node_role_policy_name: "describe_instances"
-#     loop_control:
-#       loop_var: openshift_aws_node_group
-#     with_items: "{{ openshift_aws_node_groups }}"
-#   - include_role:
-#       name: openshift_aws
-#       tasks_from:  iam_role
-#     vars:
-#       l_node_group_config: "{{ openshift_aws_master_group_config }}"
-#       openshift_aws_iam_master_role_name: "openshift_master_launch_instances"
-#       openshift_aws_iam_master_role_policy_name: "launch_instances"
-#       openshift_aws_iam_master_role_policy_json: "{{ lookup('template', 'launchinstances.json.j2') }}"
-#     loop_control:
-#       loop_var: openshift_aws_node_group
-#     with_items: "{{ openshift_aws_master_group }}"
+- name: Create iam roles
+  hosts: localhost
+  roles:
+  - openshift_aws
+  tasks:
+  - include_role:
+      name: openshift_aws
+      tasks_from: iam_role
+    vars:
+      l_node_group_config: "{{ openshift_aws_node_group_config }}"
+      openshift_aws_iam_node_role_name: "openshift_node_describe_instances_{{ openshift_aws_clusterid }}"
+      openshift_aws_iam_node_role_policy_json: "{{ lookup('file', 'describeinstances.json') }}"
+      openshift_aws_iam_node_role_policy_name: "describe_instances"
+    loop_control:
+      loop_var: openshift_aws_node_group
+    with_items: "{{ openshift_aws_node_groups }}"
+  - include_role:
+      name: openshift_aws
+      tasks_from:  iam_role
+    vars:
+      l_node_group_config: "{{ openshift_aws_master_group_config }}"
+      openshift_aws_iam_master_role_name: "openshift_master_launch_instances_{{ openshift_aws_clusterid }}"
+      openshift_aws_iam_master_role_policy_name: "launch_instances"
+      openshift_aws_iam_master_role_policy_json: "{{ lookup('template', 'launchinstances.json.j2') }}"
```

Comment:
Why does json come before name, but name before json here? Honestly @abutcher how do you miss something like this!??!?

Reply:
:construction: PEOPLE WORKING HERE :construction: